src/persona/persona.resolver.ts

```diff
@@ -0,0 +1,42 @@
+import { Args, Mutation, Query, Resolver } from '@nestjs/graphql';
+import { UseGuards } from '@nestjs/common';
+import { GqlSessionGuard } from '../guards';
+import { ConnectPersonaInput } from '../shared';
+import { PersonaService } from './persona.service';
+import { PersonaType } from './dto/persona.dto';
+import { PersonaInput } from './input';
+import { UpdatePersonaInput } from '../shared/input/update-persona.input';
+
+@Resolver('Persona')
+export class PersonaResolver {
+  constructor(private readonly personaService: PersonaService) {}
+
+  @Query(() => [PersonaType])
```

Comment:
is @Query(() => [PersonaType]) the same as @Query(returns => PersonaType)?

Reply:
I think - yes